.pipelines/e2e.yml

```diff
@@ -48,9 +48,12 @@ jobs:
       - template: ./templates/template-clean-e2e-db.yml
         parameters:
           workingDirectory: ${{ variables.modulePath }}
+        condition: always()
```

Comment:
e2e.yml can run against INT, and in that case there's no infrastructure to clean up.  These templates already have `condition:` cases inside them which should do the right thing; these should not be necessary.